llvm/unittests/Support/ErrorTest.cpp

```diff
@@ -738,17 +738,25 @@ TEST(Error, ErrorCodeConversions) {
 
 // Test that error messages work.
 TEST(Error, ErrorMessage) {
-  EXPECT_EQ(toString(Error::success()), "");
```

Comment:
I'd want to keep this, just to have one example that doesn't go through toStringWithoutConsuming or visitErrors.